src/hardware/usrp_iface.py

```diff
@@ -310,6 +313,7 @@ def acquire_samples(self, n, nskip=0, retries=5):  # -> np.ndarray:
             else:
                 nsamps = n + nskip
 
+            self.capture_time = utils.get_datetime_str_now()
```

Comment:
For this `time`, we may eventually want microsecond and even nanosecond resolution. But this will also include an entirely different `acquire_samples` function, so I believe this is good here (we can update how we store this `time` when we have the ability to get a time accurate to that level).